backend/controllers/admin-controller.js

```diff
@@ -98,6 +99,70 @@ async function getPatientsDoctors(req, res) {
     res.status(200).send(doctors);
 }
 
+async function getPatientsDoctorsSimple(req, res) {
```

Comment:
Nicely done ! There's 2 functions which get the patients assigned to the doctor, are you using both ? If not, I think you should just remove the one you're not using and rename the one you're using correspondingly, well done !

Reply:
I didnt know if it was being used somewhere else , so i just created an alternate one. I'll remove it